### DISCUSSION TITLE
OMI glTF Working Group Meeting 2023-01-26

### DISCUSSION THREAD

**Discussion by lyuma** [omigroup/gltf-extensions#138] [General]
This meeting is on 1/26/2023 at 10:30 PM (UTC) / 2:30 PM (PST) in the [OMI Discord](https://discord.gg/NJtT9grz5E) within the #omi-weekly-meeting channel. During the meeting, we will be using the #omi-gltf-extensions channel to manage a speaker queue, post links, and for any sidebar discussions.

To be notified of this meeting and others, subscribe to the [OMI Meetings and Events Calendar](https://calendar.google.com/calendar/u/1?cid=Y18wZHB1Z2Y5ZjgzZXE0cWVrbWI2b21xYmptZ0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t) or add yourself to the @omi-gltf-subgroup role in the #roles channel of the OMI Discord.

## Agenda:
* We discussed https://openmv.org (Open Metaverse Foundation) and how we can synergize with their efforts. Should we make a showcase of our projects to open up a broader discussion?
* jin - proposals how to make our specs and work more accessible to others. showcase gltf models
* Review pull requests https://github.com/omigroup/gltf-extensions/pulls
* [EXT_skin_humanoid](https://github.com/takahirox/EXT_skin_humanoid)
* [OMI_personality proposal #133](https://github.com/omigroup/gltf-extensions/pull/133) - may postpone because ant couldn't make it this week.
* [OMI_link](https://github.com/omigroup/gltf-extensions/tree/main/extensions/2.0/OMI_link)
* [OMI_spawn_point](https://github.com/omigroup/gltf-extensions/tree/main/extensions/2.0/OMI_spawn_point)
* Review [MSFT_physics](https://github.com/eoineoineoin/glTF_Physics)
  * Continue discussion from last week: [OMI glTF Working Group Meeting 12/15/2022 #128 (comment)](https://github.com/omigroup/gltf-extensions/discussions/128#discussioncomment-4414994)
  * [Replace `isKinematic` with a string enum for the body type eoineoineoin/glTF_Physics#4](https://github.com/eoineoineoin/glTF_Physics/issues/4)
  * [Remove the `centerOfMass` property eoineoineoin/glTF_Physics#5](https://github.com/eoineoineoin/glTF_Physics/issues/5)
  * [Remove the `isHollow` parameter from collider shapes eoineoineoin/glTF_Physics#6](https://github.com/eoineoineoin/glTF_Physics/issues/6)
  * [Define explicit units for all relevant properties eoineoineoin/glTF_Physics#7](https://github.com/eoineoineoin/glTF_Physics/issues/7)
* Licensing Extension https://twitter.com/superhoge/status/1596757861882740736
* [KHR_audio / Khronos feedback](https://github.com/KhronosGroup/glTF/pull/2137)
  
  * [gltf-sample-assets](https://github.com/KhronosGroup/glTF-Sample-Models/pull/360)
* [Third Room Extensions](https://github.com/matrix-org/thirdroom/pull/469)
* [glXF](https://github.com/KhronosGroup/glXF/blob/main/specification/2.0/README.md)
* Show and tell
* Create next weeks agenda

To propose another item for the agenda, comment below (preferably before the meeting).

**Comment by aaronfranke** [comment]
Discussion highlights in the first 30 minutes of the chat:
* We want to work together with OMF and any other standards groups, not against them.
* The OMF approach is to focus on taking the data that companies would otherwise put in a proprietary format and put that data in GLTF and as a result it's only ensured to work good in that context but can be technically readable in others, as opposed to our focus of trying to make general-purpose standards that work for many use cases.
* The OMF approach is an easier ask for companies because it's essentially giving them full control over the data.
* If multiple companies put their data in an open format, it's an easier job for us at OMI to look at what they have in common and build a standard from that.

**Comment by lyuma** [comment]
As a working group, I believe there to be interest in members of OMI in adapting the [EXT_skeleton_humanoid](https://github.com/takahirox/EXT_skeleton_humanoid) to serve as an animation format for existing VRM 0.0 and VRM 1.0 models which do not explicitly implement the still-WIP EXT_skeleton_humanoid extension.

So part of the work OMI may work on is to define a workflow for applying EXT_skeleton_humanoid animations to *existing* VRM 0.0 and 1.0 models.
